crates/oxc_traverse/scripts/lib/walk.mjs

```diff
@@ -75,7 +75,9 @@ function generateWalkForStruct(type, types) {
 
   const { scopeArgs } = type;
   /** @type {Field | undefined} */
-  let scopeEnterField;
+  let scopeEnterField,
+    /** @type {Field | undefined} */
+    exitScopeField;
```

Comment:
Could you name this var `scopeExitField` for consistency with `scopeEnterField`?